esphome/components/oxt_dimmer/light.py

```diff
@@ -0,0 +1,68 @@
+import esphome.codegen as cg
+import esphome.config_validation as cv
+from esphome import pins
+from esphome.components import light, uart
+from esphome.const import (
+    CONF_CHANNELS,
+    CONF_GAMMA_CORRECT,
+    CONF_OUTPUT_ID,
+    CONF_MIN_VALUE,
+    CONF_MAX_VALUE,
+    CONF_SENSING_PIN,
+)
+
+
+CODEOWNERS = ["@michau-krakow"]
+DEPENDENCIES = ["uart", "light"]
```

Comment:
Not required for light platform files
```suggestion
DEPENDENCIES = ["uart"]
```